modules/agent-installer-architectures.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="agent-install-networking_{context}"]
+= Optional: Verifying the architecture of an Agent-based installer cluster
+
+After installing an {product-title} cluster using the Agent-based Installer, you can check the architecture of the cluster.
+
+.Prerequisites: 
+
+* You installed the {oc-first}.
+* You have the `openshift-install` binary.
+
+.Procedure: 
+
+. Log in to the {oc-first}.
+
+. Check your release payload by running the following command: 
+[source,terminal]
```

Comment:
The formatting you're using is working in the preview, but the typical syntax is:
+
[source,terminal]
----
$ command
----
I would check the Customer Portal (and docs.redhat.com) after publication to make sure your alt syntax works there.

Reply:
Thanks @ShaunaDiaz ! :)